cmd/nerdctl/image_convert.go

```diff
@@ -387,3 +484,34 @@ func imageConvertShellComplete(cmd *cobra.Command, args []string, toComplete str
 	// show image names
 	return shellCompleteImageNames(cmd)
 }
+
+type imageInfo struct {
+	Name   string // image name
+	Digest string // image target digest (i.e. index digest or manifest digest)
```

Comment:
Can we just name and digest into a single `Image` string concatenated with `@`?

Reply:
Thank you for the review. Fixed the struct format.

```console
# nerdctl image convert --format=json --oci --estargz --estargz-external-toc ghcr.io/stargz-containers/ubuntu:22.04 registry2:5000/ubuntu:22.04-ex
{
    "image": "registry2:5000/ubuntu:22.04-ex@sha256:3059dd5d9c404344e0b7c43d9782de8cae908531897262b7772103a0b585bbee",
    "extra_images": [
        "registry2:5000/ubuntu:22.04-ex-esgztoc@sha256:18c042b6eb8b208a8b97c88ad6d5186893141cb4ff43fab8c9bbb936eba8ae89"
    ]
}
# nerdctl image convert --oci --estargz --estargz-external-toc ghcr.io/stargz-containers/ubuntu:22.04 registry2:5000/ubuntu:22.04-ex
INFO[0006] Extra image(0) registry2:5000/ubuntu:22.04-ex-esgztoc 
sha256:3059dd5d9c404344e0b7c43d9782de8cae908531897262b7772103a0b585bbee
```